simapp/simd/cmd/root.go

```diff
@@ -123,6 +123,7 @@ func initAppConfig() (string, interface{}) {
 	//
 	// In simapp, we set the min gas prices to 0.
 	srvCfg.MinGasPrices = "0stake"
+	// srvCfg.BaseConfig.IAVLDisableFastNode = true // disable fastnode by default
```

Comment:
Why did you comment out this line?

Reply:
I just copied the code from cosmos-sdk. I wanted to delete it, but I left it because conflicts would occur in the bump up process of cosmos-sdk later.